modules/ExchangeApi.py

```diff
@@ -0,0 +1,114 @@
+'''
```

Comment:
I love this approach, makes a lot of sense and means people will have a hard time adding new exchanges incorrectly.

The only thing that's missing for the loan side is toggleAutoRenew, but we removed support for that from the bot anyway. 

Maybe there should be a `__repr__` too? I imagine in the future the bot will run multiple exchanges in one process, that would make it a bit easier to debug.